packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -0,0 +1,47 @@
+import {Component, Element, h} from '@stencil/core';
+import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rounded.svg';
+import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
+
+@Component({
+  tag: 'atomic-segmented-facet-scrollable',
+  styleUrl: 'atomic-segmented-facet-scrollable.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacetScrollable {
+  @Element() private host!: HTMLElement;
+  // @State() children: Array<any> = [];
+
+  connectedCallback() {
+    // this.children = Array.from(this.host.children);
+    // console.log(this.children);
+
+    // const shadowRoot =
+    //   this.host.shadowRoot === null
+    //     ? this.host.attachShadow({mode: 'open'})
+    //     : this.host.shadowRoot;
+
+    console.log(this.host.shadowRoot);
+  }
+
+  render() {
+    return (
+      <div class="flex">
+        <div class="border border-neutral bg-background text-on-background hover:bg-neutral-light focus-visible:bg-neutral-light no-outline rounded">
+          <atomic-icon
```

Comment:
When adding a label and especially a label with 2 or more words the arrow buttons aspect ratio changes. I need to find a way to keep it constant and also adjust the size of the arrow icon as well.

Reply:
The segmented facet have a height of 35px without a label but of 41px with a label. We should set a static height on the facet and use flex to align the items vertically.

After that it should be easy to use a good scrollable aspect ratio

Reply:
I fixed some of the issues ! I will implement your solution as well. Thanks !